### PR TITLE
Add basic command line

### PR DESCRIPTION
**Pull request by jorisvandenbossche**

### CODE REVIEW

**Comment by jorisvandenbossche** [comment]
@glemaitre ready for review. I added a `convert` sub-command, this is more explicit in what it does and keeps a bit more flexibility future-wise

File: specio/cli.py [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+             'as input with different extension.')
+
+    args = parser.parse_args()
+
+    if args.sub == 'convert':
+        spectrum = specread(args.filepath)
+        if args.output:
+            output_path = args.output[0]
+        else:
+            output_path = os.path.splitext(args.filepath)[0] + '.csv'
```
Comment (glemaitre) [inline specio/cli.py]: We will probably need to expand the name if we have a wildcard. We can call this internal function:

https://github.com/paris-saclay-cds/specio/blob/d28ddce027a1d272affade6d9392fd04b66e62f4/specio/core/functions.py#L122-L144

**Comment by glemaitre** [comment]
@jorisvandenbossche Good for review

File: specio/cli.py [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+    args = parser.parse_args()
+
+    if args.sub == 'convert':
+        filenames = _validate_filenames(args.filepath)
+        spectrum = specread(filenames)
+
+        # case that we could not merge the spectra together
+        if isinstance(spectrum, list):
+            if args.output:
+                output_basename = args.output[0]
```
Comment (jorisvandenbossche) [inline specio/cli.py]: Do we need to remove the extension? (if somebody did `-o spectra.csv`

**Comment by glemaitre** [comment]
Good point. Solved.

**Comment by glemaitre** [comment]
@jorisvandenbossche Shall I merge?

File: doc/api.rst [hunk 3]
```diff
@@ -64,6 +64,33 @@ For more information, you can check the full API documentation above.
...
+``specio`` provides a command line interface to make basic data manipulation.
+
+Conversion to CSV
+-----------------
+
+Any format can be exported using the ``specio`` command line::
+
+  specio convert input_file.spc output_file.csv
+
+``convert`` receive the input and output arguments. It is also possible to the
```
Comment (jorisvandenbossche) [inline doc/api.rst]: to "omit" the ?

File: doc/api.rst [hunk 4]
```diff
@@ -64,6 +64,33 @@ For more information, you can check the full API documentation above.
...
+output argument and an automatic output file will be generated depending of the
+input file name::
+
+  specio convert input_file.spc
+
+As with :func:`specio.specread`, an input can get a name with a wildcard::
+
+  specio convert "*.spc"
+
+In addition, this is also possible to move the tolerance up to aggregate
```
Comment (jorisvandenbossche) [inline doc/api.rst]: "Move the tolerance up" is a bi strange wording. "Increase", or just "change" or "specify" ?

**Pull request merged**